src/uwtools/cli/atparse_to_jinja2.py

```diff
@@ -53,8 +55,23 @@ def parse_args(args: List[str]) -> Namespace:
         help="Path to new Jinja2 template",
         metavar="FILE",
     )
+    optional.add_argument(
+        "-q",
+        "--quiet",
+        action="store_true",
+        help="Print no logging messages",
+    )
+    optional.add_argument(
+        "-v",
+        "--verbose",
+        action="store_true",
+        help="Print all logging messages.",
+    )
     parsed = parser.parse_args(args)
     if not parsed.dry_run and not parsed.outfile:
         print("Specify either --dry-run or --outfile", file=sys.stderr)
         sys.exit(1)
+    if parsed.quiet and parsed.verbose:
+        print("Options --quiet and --verbose may not be used together", file=sys.stderr)
```

Comment:
Note here and below, in CLI code, that `print(..., file=sys.stderr)` is used to emit messages to `stdout` since logging has not been set up yet, and cannot be until argument parsing is complete and the status of the `--quiet` and `--verbose` switches is known.

Reply:
I'm a bit confused here. Would these messages not be emitted to `stderr`? Is that a typo or black magic?

Reply:
Yes, to `stderr`. Is it the `file=` part that is surprising, i.e. `stderr` is not a file?